delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/model/FileState.scala

```diff
@@ -1,5 +1,6 @@
 package ch.epfl.bluebrain.nexus.delta.plugins.storage.files.model
 
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.model.ResourcesSearchParams.FileUserMetadata
```

Comment:
It should not use an import from the ES plugin

Reply:
Interesting. You are right, I see two possibilities:
- Put everything in FileAttributes
- Make a class in non-plugin code to represent the data

I think I'm leaning to using FileAttributes, what do you think @dantb @imsdu ?

Reply:
`FileUserMetadata` feels like a file thingy in the way it's supported by the API, so that seems reasonable to me